pub_sub/java/sdk/README.md

```diff
@@ -0,0 +1,60 @@
+# Dapr pub/sub
+
+In this quickstart, you'll create a publisher microservice and a subscriber microservice to demonstrate how Dapr enables a publish-subcribe pattern. The publisher will generate messages of a specific topic, while subscribers will listen for messages of specific topics. See [Why Pub-Sub](#why-pub-sub) to understand when this pattern might be a good choice for your software architecture.
```

Comment:
```suggestion
In this quickstart, you'll create a publisher microservice and a subscriber microservice that demonstrates how Dapr enables a publish-subscribe pattern. The publisher will publish messages to a specific topic, while subscribers will listen for messages from a specific topic. See [Why Pub-Sub](#why-pub-sub) to understand when this pattern might be a good choice for your software architecture.
```

Reply:
We have similar wordings at other examples, So to keep it consistent I have used similar wordings.